tests/testthat/test-sanitize-filename.R

```diff
@@ -0,0 +1,52 @@
+context("sanitize-filename")
+
+test_that("File names can't have multiple dots", {
+  expect_equal(sanitizeFileName("../baz.jpg", "default"), ".baz.jpg")
```

Comment:
Probably multiple dots should be checked after whitelist filter has been applied? In current configuration,"./.bar.csv" will be sanitized to ..bar.csv